lib/users/routes.js

```diff
@@ -71,16 +92,126 @@ TODO: inject content into form
 @function
 @name /users/edit/{{id}}
 */
-router.get('/edit/:id', (req, res) => {
-  // this render should populate the form
-  res.render('users/add', {
-    pageTitle: `${config.users.usersHome.title} :: Edit User`,
+router.get('/edit/:id',
+  loginMiddleware({
+    'role': 'superadmin',
+    'redirectTo': '/login',
+    'setReturnTo': '/users/add',
+  }),
+  (req, res, next) => {
+    req.session.editUserId = req.params.id; // eslint-disable-line no-param-reassign
+
+    database('users').where('id', req.params.id)
+    .then(rows => {
```

Comment:
Please be consistent with where you put the first `.then`. It should go after the call, not on the next line